### ISSUE TITLE
Find an alternative method for locating nearby Pokemons.

### ISSUE THREAD

**Issue by mayank26saxena** [mayank26saxena/PokeBot#1]
PokeVision API has been discontinued. Look for an alternative method for finding nearby Pokemons.

**Comment by Naramsim** [comment]
Are you using pokevision or skiplagged?

**Comment by mayank26saxena** [comment]
I was using skiplagged.

**Comment by Naramsim** [comment]
Does it work now? I tried the bot but it seems not working.

**Comment by mayank26saxena** [comment]
No, since the skiplagged API has stopped working, the bot also doesn't work anymore. If you are aware of any other API which I can use then let me know!
